docs/clients/p4rt-ctl.rst

```diff
@@ -319,7 +319,61 @@ Details about the supported commands
     ``COUNTER_FLOW``: Refers to the counter ID (generated ID by p4c, refer to bfrt.json file)
     and the counter table index. Format "counter_id=<number>,index=<number>"
 
-16. Get flow dump entries ::
+
+16. Add a rule for direct counter into the forwarding pipeline::
+
+    $ p4rt-ctl add-entry SWITCH TABLE FLOW
+    $ Example::
+    [Ingress/Rx]
```

Comment:
Please check rst formatting here:
<img width="902" alt="image" src="https://github.com/ipdk-io/networking-recipe/assets/22621220/439e6cb5-a42f-4054-9da5-c38fa546290e">

Reply:
Same comment for all examples